### PR TITLE
feedback about .NET tracing setup instructions

### PR DESCRIPTION
**Pull request by kayayarai**
### What does this PR do?
Adds Agent install/config instruction to .NET tracing setup instructions

### Motivation
feedback from Wan

### Preview

https://docs-staging.datadoghq.com/kari/dot-net-agent-config/tracing/setup_overview/setup/dotnet-framework
https://docs-staging.datadoghq.com/kari/dot-net-agent-config/tracing/setup_overview/setup/dotnet-core

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/setup_overview/setup/dotnet-core.md [hunk 1]
```diff
@@ -278,6 +278,10 @@ Once this is set, verify that the environment variables were set with `systemctl
 
 **Note:** The .NET runtime tries to load a profiler into _any_ .NET process that is started with these environment variables set. You should limit instrumentation only to the applications that need to be traced. **We do not recommend setting these environment variables globally as this causes _all_ .NET processes on the host to load the profiler.**
 
+### Configure the Datadog Agent for APM
+
+Install and configure the Datadog Agent to receive traces from your now instrumented application. By default the Datadog Agent is enabled in your `datadog.yaml` file under `apm_enabled: true` and listens for trace traffic at `localhost:8126`. For containerized environments, follow the links above to enable trace collection within the Datadog Agent.
```
Comment (sarina-dd) [inline content/en/tracing/setup_overview/setup/dotnet-core.md:283]:
```suggestion
Install and configure the Datadog Agent to receive traces from your instrumented application. By default the Datadog Agent is enabled in your `datadog.yaml` file under `apm_enabled: true` and listens for trace traffic at `localhost:8126`. For containerized environments, follow the links above to enable trace collection within the Datadog Agent.
```

File: content/en/tracing/setup_overview/setup/dotnet-framework.md [hunk 2]
```diff
@@ -113,6 +113,10 @@ rem Start application
 example.exe
 ```
 
+## Configure the Datadog Agent for APM
+
+Install and configure the Datadog Agent to receive traces from your now instrumented application. By default the Datadog Agent is enabled in your `datadog.yaml` file under `apm_enabled: true` and listens for trace traffic at `localhost:8126`.
```
Comment (sarina-dd) [inline content/en/tracing/setup_overview/setup/dotnet-framework.md:118]:
```suggestion
Install and configure the Datadog Agent to receive traces from your instrumented application. By default the Datadog Agent is enabled in your `datadog.yaml` file under `apm_enabled: true` and listens for trace traffic at `localhost:8126`. 
```

**Review by sarina-dd — APPROVED** [review]
Small suggestions, LGTM 👍

**Comment by zacharycmontoya** [comment]
@kayayarai thanks for bringing up this PR. I think before merging this PR, @jaycdave88 and I need to add more complete containerized and non-containerized setup instructions to the .NET pages

**Comment by jtappa** [comment]
Closing to defer to #9375

**Pull request closed without merging**